soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -64,6 +64,7 @@ config SOC_NRF5340_CPUAPP
 	select HAS_HW_NRF_USBREG
 	select HAS_HW_NRF_WDT0
 	select HAS_HW_NRF_WDT1
+	select HAS_PM
```

Comment:
AFAICT this doesn't deal with the fact that this is true only for the app core.  I'm not sure we can override it then, because select is pretty permanent.  In another recent PR we agreed the upshot of https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#select-pitfalls is that select is "evil".

Reply:
Or since I notice this is in the CPUAPP block, maybe we wouldn't need to override it.  Still, if the effect of this is that you can't disable `CONFIG_PM`, that'd be a bad thing.  I don't know if it's true, though.  Kconfig is unintuitive at best.

Reply:
the select case here is not the same that is described in that documentation. Here `HAS_PM` option is hidden and does not select anything when selected. This pattern very used across Zephyr.

Reply:
Agreed.